index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/css/normalize.css">
+    <link rel="stylesheet" href="/css/styles.css">
+    <title>PC Store</title>
+</head>
+
+<body>
+    <div class="bar">
+        <header>
+            <div class="logo">
+                <img class="logo-image" src="/img/logo.png" alt="">
```

Comment:
I would recommend that for the logo it would be better to add an "a href" tag as a home button. that when someone clicks on it it will send you to the home page.